l10n_nl_tax_statement/views/l10n_nl_vat_statement_view.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017-2019 Onestein (<https://www.onestein.eu>)
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record id="view_l10n_nl_vat_report_form" model="ir.ui.view">
+        <field name="model">l10n.nl.vat.statement</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <field name="state" widget="statusbar" statusbar_visible="draft,posted,final"/>
+                </header>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button name="statement_update" string="Update" states="draft" type="object" class="oe_stat_button" icon="fa-repeat"/>
+                        <button name="post" string="Post" states="draft" type="object" class="oe_stat_button" icon="fa-arrow-right text-success"/>
+                        <button name="reset" string="Reset to Draft" states="posted" type="object" class="oe_stat_button" icon="fa-arrow-left text-success"/>
+                        <button name="finalize" string="Finalize" states="posted" type="object" class="oe_stat_button" icon="fa-stop-circle-o text-success" confirm="If you confirm, it will be not possible to modify this Statement or reset it back to draft anymore. Do you confirm?"/>
+                    </div>
+                    <label for="name"/>
+                    <h1>
+                        <field name="name"/>
+                    </h1>
+                    <group>
+                        <group name="tax_report">
+                            <field name="company_id" options="{'no_create': True}" groups="base.group_multi_company"/>
+                            <field name="parent_id" attrs="{'invisible': [('parent_id', '=', False)]}" groups="base.group_multi_company"/>
+                            <field name="display_multicompany_fiscal_unit" invisible="1" groups="base.group_multi_company"/>
+                            <field name="multicompany_fiscal_unit" attrs="{'readonly': [('state','in',['posted','final'])],'invisible': [('display_multicompany_fiscal_unit', '=', False)]}" groups="base.group_multi_company"/>
```

Comment:
Only visible if you select all companies in the multi-company switcher. Shouldn't it also work if we just select the parent company?

Reply:
It works based on the company selected in the statement, regardless of the companies you selected in the multi-company switcher.

Reply:
Yes but you can only select the companies of all companies which are selected in the switcher.

Reply:
Yes, seems correct to me. You cannot work on a company if not selected. Is the standard Odoo behavior.